packages/core/admin/ee/admin/pages/SettingsPage/utils/customRoutes.js

```diff
@@ -1,9 +1,11 @@
-import SingleSignOn from '../SingleSignOn';
-
 const ssoRoutes = strapi.features.isEnabled(strapi.features.SSO)
   ? [
       {
-        Component: SingleSignOn,
+        Component: async () => {
+          const component = await import(/* webpackChunkName: "sso" */ '../SingleSignOn');
```

Comment:
I think you've named the "pages" with a suffix of "-pages". Isn't this a settings page?

Reply:
You are right, it is 👍🏽